src/ReferenceFEs/StrangQuadratures.jl

```diff
@@ -72,7 +72,7 @@ function _strang_quad_tri(degree)
     et2 = 0.445948490915965
     ez2 = 0.445948490915965
     a = 0.054975870996713638
-    b = 0.1116907969117165    
+    b = 0.1116907969117165
```

Comment:
see https://github.com/gridap/Gridap.jl/pull/840/files#r999428126. `b` is `Float64`

Reply:
Yes, implicit conversion I felt (like commented above) to be more efficient than converting the whole vector of `Float64`. A tedious way would be to wrap `fptype` around all these values, I feel the current way and this way internally do the same thing of converting `Float64` values to `Float32`.